suite/tests/CMakeLists.txt

```diff
@@ -2948,6 +2948,7 @@ else (UNIX)
     append_link_flags(security-win32.except-execution "/dynamicbase:no")
 
     tobuild(security-win32.except-guard security-win32/except-guard.c)
+    tobuild(security-win32.except-int2d security-win32/except-int2d.c)
```

Comment:
So here it will not be tested on 64-bit -- was that the intention?  If it works for 64-bit let's get it out of this NOT X64 block.